hack/quickstart-setup.sh

```diff
@@ -504,13 +506,26 @@ info "Deploying Kuadrant sample configuration..."
 kubectl -n ${KUADRANT_NAMESPACE} apply -f ${KUADRANT_REPO_RAW}/config/samples/kuadrant_v1beta1_kuadrant.yaml
 success "Kuadrant sample configuration deployed."
 
+# Install thanos on hub cluster
+if [ "$HUB" -eq 1 ]; then
+  info "Installing thanos in ${KUADRANT_CLUSTER_NAME}... (as hub cluster)"
+  kubectl apply -k ${KUADARNT_THANOS_KUSTOMIZATION}
+  success "thanos installed successfully."
+fi
+
 # Install observability stack
 info "Installing observability stack in ${KUADRANT_CLUSTER_NAME}..."
 kubectl kustomize ${KUADARNT_OBSERVABILITY_KUSTOMIZATION} | docker run --rm -i ryane/kfilt -i kind=CustomResourceDefinition | kubectl apply --server-side -f -
```

Comment:
The script fails at this point for me saying docker command not found. I do have a alias but maybe the pipe is causing weird things with it. Its not a change from this pr that's causing the issue but Im thinking we should use the CONTAINER_RUNTIME_BIN env from the top of the script instead

Reply:
Agreed.
Pushed up a fix for that